Makefile

```diff
@@ -169,9 +195,3 @@ clean:
 	@rm -rf .$/build
 
 .PHONY: test build-source
-
-apply-patches:
-	@echo "$(CYAN) Applying patches...$(RESET)"
-
-	patch -p1 < ./resource/macos/macPackager-patch.diff
-	patch -p1 < ./resource/macos/scheme-patch.diff 
```

Comment:
There are some repetitive elements here I'd like to consider undoing (unless we have concrete issues triggering the changes).

 - we're removing `$/` in favor of `/` which at least used to prevent breakage in Windows. when running in WSL and some other environments I know this won't be as much of a factor, but if that's the case it's probably better to leave the changes out of this patch and do so entirely in one go in a PR like "Remove no-longer-necessary `$/` substitution"

 - Maybe green looks better than cyan but I'm not seeing a warrant for changing it or a need to do so as part of the Electron upgrade. If we don't like it we could more appropriately address that in its own PR "Change the color of messages to more closely indicate status" or something like that. Here it's conflating the responsibility of what's necessary for the change and what's an unrelated styling issue.

Reply:
@dmsnell 

- The use of `$/` actually doesn't work on Windows, so this is a necessary change (see comment above) - feel free to verify/test this locally. IMO we shouldn't have these platform-specific snippets if we don't need them.

- The change to green is an artifact of the fact that until a few days ago, we thought we would be going with AppVeyor for our CI solution on Windows - I found it much easier to distinguish the Green from Cyan in contrast to AppVeyor's choice of background color, which helped make navigating the build logs much easier. That being said, since we're going with Circle I'm happy to revert to the original color and minimize the overall diff.

Reply:
Update: Reverted to usage of cyan in 7e0742a